submissions/SkirnevskyiAleksandr/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,185 @@
+    <!--dropdown-menu-->                
+                <ul class="DropDownMenu">
+
+                    <li class="DropDownMenu-item">
+                        <a class="DropDownMenu-item" href="#">
+
+                            <img src="./icons/blogs.png" class="popupItem" alt="Blogs">
+                            <span>Blogs</span>
+
+                        </a>
+                    </li>
+                    <li class="DropDownMenu-item">
+                        <a class="DropDownMenu-item" href="#">
+
+                            <img src="./icons/calendar.png" class="popupItem" alt="calendar"><span>Calendar</span>
+
+                        </a>
+                    </li>
+                    <li class="DropDownMenu-item">
+                        <a class="DropDownMenu-item" href="#">
+                            <f>
+                                <img src="./icons/chat.png" class="popupItem" alt="chat"><span>Chat</span>
+
+                        </a>
+                    </li>
+                    <li class="DropDownMenu-item">
+                        <a class="DropDownMenu-item" href="#">
+
+                            <img src="./icons/classroom.png" class="popupItem" alt="classroom"><span>Classroom</span>
+
+                        </a>
+                    </li>
+                    <li class="DropDownMenu-item">
+                        <a class="DropDownMenu-item" href="#">
+
+                            <img src="./icons/collection.png" class="popupItem" alt="collection"><span>Collection</span>
+
+                        </a>
+                    </li>
+                    <li class="DropDownMenu-item">
+                        <a class="DropDownMenu-item" href="#">
+
+                            <img src="./icons/documents.png" class="popupItem" alt="documents"><span>documents</span>
+
+                        </a>
+                    </li>
+                    <li class="DropDownMenu-item">
+                        <a class="DropDownMenu-item" href="#">
+
+                            <img src="./icons/files.png" class="popupItem" alt="files"><span>Files</span>
+
+                        </a>
+                    </li>
+                    <li class="DropDownMenu-item">
+                        <a class="DropDownMenu-item" href="#">
+
+                            <img src="./icons/mail.png" class="popupItem" alt="mail"><span>Mail</span>
+
+                        </a>
+                    </li>
+                    <li class="DropDownMenu-item">
+                        <a class="DropDownMenu-item" href="#">
+
+                            <img src="./icons/maps.png" class="popupItem" alt="maps"><span>Maps</span>
+
+                        </a>
+                    </li>
+    <!--btn-more-->                
+                    <li class="btn">
+                        <input id="BtnMore" class="BtnMore" type="checkbox">
+                        <label for="BtnMore" class="BtnMore-Label">More</label>
```

Comment:
camelCase style and  hyphen for class names are not desirable for html-markup.  It would be better to use the only hyphen (more-label)  or underscore (more_label).  Could you please fix it for all class names?